docs/basics/ErrorHandling.md

```diff
@@ -71,3 +71,14 @@ function* fetchProducts() {
     yield put({ type: 'PRODUCTS_REQUEST_FAILED', error })
 }
 ```
+
+## onError hook
+Errors in forked tasks [bubble up to their parents](../api/README.md#error-propagation)
+until it is caught or reaches the root saga.
+If an error propagates to the root saga the whole saga tree is already **terminated**. The preferred approach, in this case, to use [onError hook](../api/README.md#error-propagation#createsagamiddlewareoptions) to report an exception, inform a user about the problem and gracefully terminate your app.
+
+Why I cannot use `onError` hook as a global error handler?
+Usually, there is no one-size-fits-all solution, as exceptions are context dependent. Consider `onError` hook as the last resort that helps you to handle **unexpected** errors.
+
+Why if I don't want an error to bubble?
```

Comment:
Typo: why -> what

Reply:
@levenleven would you like to prepare a PR fixing this?

Reply:
Sure, no problem